spec/std/ini_spec.cr

```diff
@@ -3,14 +3,38 @@ require "ini"
 
 describe "INI" do
   describe "parse from string" do
+    it "fails on malformed section" do
+      expect_raises(INI::ParseException, /unterminated section/) do
```

Comment:
You could use a string here and in the following examples, no need for a regex (the point of this PR is reducing them^^).